### PR TITLE
More In Depth Explanation

### PR DESCRIPTION
**Pull request by Santa-Bunny**
I added more photos and step by step to set up a dev environment. I still need to explain what to hook up and to make a Bill of Materials for each type of dev system, but this will at least give a starting point.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -4,13 +4,167 @@
...
+
+The final Magic Wifi product can successfully send Ethernet frames over both Wifi and LoRa. However, it remains very susceptible to interference, both on Wifi and LoRa which led to our live demo not working properly at the expo. Even at our later instructor demo, this remained an issue in campus buildings so we had to show a recording of certain features.  At the beginning we had concerns about sending frames over LoRa due to the low bandwidth and need for subframing due to a small maximum LoRa frame size.
```
Comment (bluebandit21) [inline docs/README.md]: Lots of stuff here that makes sense in the context of where it was copied from but probably doesn't belong in this repo :)
(Interference bits certainly relevant! Maybe just not the context of where we encountered them, etc)

File: docs/README.md [hunk 2]
```diff
@@ -4,13 +4,167 @@
...
+![image not found](images/Top_PCB_V1_2.png)
+Fig: An image of Version 1.2 of the MagicWiFi PCB, the first working version with modifications.
+
+If you a developing on a PCB version of MagicWiFi, to Flash binaries built for MSP430 you will need the [MSP430 Flasher](https://www.ti.com/tool/MSP430-FLASHER), and a TI JTAG flashing device to flash firmware to the MSP430.
+
+All necessary Components are listed in [docs/BillsOfMaterials/PCB-BOM](docs/BillsOfMaterials/PCB-BOM.md)
+
+### Major Issues
+
+One major issue is At time of writing Jan 10, 2024. The ATWINC1500 modules for sale on DigiKey have an earlier version of firmware than is necessary. On the module there is not a method of flashing new firmware. This was overcome in the development process by desoldering the ATWINC1500 from the module and soldering on an ATWINC1500 from a dev board that could have the firmware updated. A new method must be decided upon as this requires more specialized tools and is no user friendly.
```
Comment (bluebandit21) [inline docs/README.md]: > and is no user friendly.
not ;)

File: docs/README.md [hunk 3]
```diff
@@ -4,13 +4,167 @@
...
+
```
Comment (bluebandit21) [inline docs/README.md]: I know why it was there in the report, but I don't think this belongs here, right?
It seems like a worse version than someone looking at the actual header files, and seems almost guaranteed to immediately get out of sync with the originals :)

**Review by bluebandit21 — CHANGES_REQUESTED** [review]
Looks mostly good, thank you so much for writing this up Santa!

Made a few minor comments, but other than that I think this is all solid and incredibly helpful :D

File: docs/README.md (hunk 1 above)
Comment (Santa-Bunny) [reply docs/README.md]: Was copy pasting a lot and must have graabed a little extra on accident, good catch

File: docs/README.md (hunk 3 above)
Comment (Santa-Bunny) [reply docs/README.md]: fair. I have to remember general audience on the system.

**Comment by Santa-Bunny** [comment]
Should be all touched up. I removed a few entire paragraphs as they were mentioned in other sections of the README in better detail and I didn't see a reason to repeat them and cause confusion and more hassle later.

**Review by Santa-Bunny — COMMENTED** [review]
should be good

**Review by bluebandit21 — APPROVED** [review]

**Comment by bluebandit21** [comment]
Fantastic, TY!

**Pull request merged**